monte-carlo/random-numbers.md

```diff
@@ -34,12 +34,9 @@ While inverse transform sampling is useful, it is sometimes inconvenient or not
 4. If $Z \le \frac{f(X)}{Mg(Y)}$, then accept the sample X.
 5. Repeat step 4 until enough samples have been drawn.
 
-## 5.2.5 Additional Resources
+## Additional Resources
 
-Bradley, T., et al. "Chapter 16-parallelization techniques for random number generators." GPU Computing Gems Emerald Edition: 231-246
-
-Gilks, Walter R., and Pascal Wild. "Adaptive rejection sampling for Gibbs sampling." Journal of the Royal Statistical Society: Series C (Applied Statistics) 41.2 (1992): 337-348.
-
-Olver, Sheehan, and Alex Townsend. "Fast inverse transform sampling in one and two dimensions." arXiv preprint arXiv:1307.1223 (2013).
-
-Tezuka, Shu. "Linear congruential generators." Uniform Random Numbers: Theory and Practice. Boston, MA: Springer US, 1995. 57-82..
\ No newline at end of file
+{cite}'bradleychapter'

```

Comment:
Almost! 

Please edit it to be like this: 
``` 
- Bradley, T., et al. "Chapter 16-parallelization techniques for random number generators." GPU Computing Gems Emerald Edition: 231-246 {cite}`bradleychapter` 
-  and so on
```
Note the type of quotation mark around "`bradleychapter`" 

You can check that is displays correctly [here](https://stattlab.github.io/atomistic-scale-simulations/preview/pr-25/molecular-dynamics/neighbor-list.html#additional-resources).